libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/idp/IDPAuthCodeActivity.kt

```diff
@@ -0,0 +1,55 @@
+package com.salesforce.androidsdk.auth.idp
+
+import android.app.Activity
+import android.os.Bundle
+import android.view.WindowManager.LayoutParams
+import android.webkit.WebSettings.LayoutAlgorithm
+import android.webkit.WebView
+import com.salesforce.androidsdk.R
+import com.salesforce.androidsdk.app.SalesforceSDKManager
+import com.salesforce.androidsdk.util.LogUtil
+import com.salesforce.androidsdk.util.SalesforceSDKLogger
+import com.salesforce.androidsdk.auth.idp.interfaces.IDPAuthCodeActivity as IDPAuthCodeActivityInterface
+
+/**
+ * Activity showing web view through which IDP gets auth code for SP app
+ */
+class IDPAuthCodeActivity : Activity(), IDPAuthCodeActivityInterface {
+    companion object {
+        val TAG: String = IDPAuthCodeActivity::class.java.simpleName
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
```

Comment:
The auth code activity.
Not much going on, setting up the web view, "attaching" itself to the IDP manager flow and giving the intent to the IDP manager so that existing handling code can do the work.